pinot-common/src/main/java/org/apache/pinot/common/utils/DataSchema.java

```diff
@@ -297,6 +297,7 @@ public DataType toDataType() {
         case TIMESTAMP:
         case TIMESTAMP_ARRAY:
           return DataType.TIMESTAMP;
+        case OBJECT:
```

Comment:
This is added to confirm the test. This shouldn't be here.

Question: to determine the `TransformResultMetadata _resultMetadata`  in `ScalarTransformFunctionWrapper` for `Object` resultType; what should we do? Like in `CaseTransformFunction` we have [calculateResultMetadata](https://github.com/apache/pinot/blob/master/pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/CaseTransformFunction.java#L102), do we have to do similar?